modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -1,6 +1,6 @@
 = Create and Configure Endpoints
 
-You can create endpoints in the host or partner page, or by using the Message Flow wizard.
+You can create endpoints in the host, or partner page or by using the Message Flow wizard.
```

Comment:
This was actually more correct in its original version:

You can create endpoints
   in the host or partner page
   or
   by using the Message Flow wizard

I think it reads just fine with no commas at all:

```suggestion
You can create endpoints in the host or partner page or by using the Message Flow wizard.
```

Reply:
OR
```suggestion
You can create endpoints in the host, on the partner page, or by using the Message Flow wizard.
```